### PR TITLE
[Typescript][Angular] Fix generated README when using apiModulePrefix

### LINKED ISSUE #7722: [BUG][TYPESCRIPT-ANGULAR] client generated README with apiModulePrefix
##### Description

When specifying the `apiModulePrefix` the generated README still says to import `ApiModule`.

If you specify a prefix (eg `PetStore`) the generated README should say to import `PetStoreApiModule`.

##### openapi-generator version

4.3.0 (also checked on master)

##### OpenAPI declaration file content or url

The petstore YAML is sufficient to reproduce it.

##### Command line used for generation

You can generate using the samples, as the generated code contains the bug.

You can use the `typescript-angular-with-prefixed-module-name-v8` sample:

```
sh bin/generate-samples.sh bin/configs/typescript-angular-with-prefixed-module-name-v8.yaml
```

##### Steps to reproduce

1. clone the openapi-generator repo
2. setup the repo (`mvn clean package`)
3. generate the sample with prefixed module: `sh bin/generate-samples.sh bin/configs/typescript-angular-with-prefixed-module-name-v8.yaml`

Expected result:
Sample readme (`samples/client/petstore/typescript-angular-v8-provided-in-root/builds/with-prefixed-module-name/README.md`) should contain `PetStoreApiModule`.

Actual result:
Sample readme contains `ApiModule`.

See the [general usage](https://github.com/OpenAPITools/openapi-generator/tree/master/samples/client/petstore/typescript-angular-v8-provided-in-root/builds/with-prefixed-module-name#general-usage) generated for the sample.

##### Suggest a fix/enhancement

I have a branch with the fix, and I can make a PR if you agree. See: https://github.com/OpenAPITools/openapi-generator/compare/master...sorohan:bugfix/typescript-angular-docs-with-module-prefix?expand=1

The thing I'm least sure about is updating the section on "Using multiple OpenAPI files / APIs / ApiModules". The advice given currently is to use:

```
import { ApiModule as OtherApiModule } from 'my-other-api-path';
```

But I think it would be better to just recommend people to use the `apiModulePrefix` option, as I assume that's what it's there for.

### PR DESCRIPTION
**Pull request by sorohan**
fixes #7722

### CODE REVIEW

File: modules/openapi-generator/src/main/resources/typescript-angular/README.mustache [hunk 1]
```diff
@@ -124,21 +124,23 @@ export class AppComponent {
...
-in order to avoid naming conflicts:
+you can avoid naming conflicts by configuring `apiModulePrefix` when generating other modules.
+
+Eg: Configuring another module using `apiModulePrefix` of `Other`:
+
 ```
-import { ApiModule } from 'my-api-path';
-import { ApiModule as OtherApiModule } from 'my-other-api-path';
+import { {{apiModuleClassName}} } from 'my-api-path';
+import { OtherApiModule } from 'my-other-api-path';
```
Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/typescript-angular/README.mustache:138]: please keep the alias example with `as`, see https://github.com/OpenAPITools/openapi-generator/issues/7722#issuecomment-709406844

File: modules/openapi-generator/src/main/resources/typescript-angular/README.mustache [hunk 2]
```diff
@@ -124,21 +124,23 @@ export class AppComponent {
...
 
-Note: The ApiModule is restricted to being instantiated once app wide.
+Note: The {{apiModuleClassName}} is restricted to being instantiated once app wide.
 This is to ensure that all services are treated as singletons.
 
 #### Using multiple OpenAPI files / APIs / ApiModules
 In order to use multiple `ApiModules` generated from different OpenAPI files,
-you can create an alias name when importing the modules
-in order to avoid naming conflicts:
+you can avoid naming conflicts by configuring `apiModulePrefix` when generating other modules.
```
Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/typescript-angular/README.mustache:132]: please reword to include the case when the prefix cannot be set:
https://github.com/OpenAPITools/openapi-generator/issues/7722#issuecomment-709406844

**Comment by macjohnny** [comment]
@sorohan thanks for your contribution

File: modules/openapi-generator/src/main/resources/typescript-angular/README.mustache (hunk 2 above)
Comment (sorohan) [reply modules/openapi-generator/src/main/resources/typescript-angular/README.mustache]: I've just reverted the wording (but changed it to use `{{apiModuleClassName}}`). I think it's best like this. Now only the sample for `with-prefixed-module-name` has changed.

File: modules/openapi-generator/src/main/resources/typescript-angular/README.mustache (hunk 1 above)
Comment (sorohan) [reply modules/openapi-generator/src/main/resources/typescript-angular/README.mustache]: Done

**Review by macjohnny — APPROVED** [review]

**Comment by macjohnny** [comment]
@sorohan please rebase onto the current master and update the readme for the customized configuration class name

**Comment by sorohan** [comment]
@macjohnny I've rebased it now. I'm not sure what update to the readme you mean. Can you clarify?

**Review by macjohnny — APPROVED** [review]
LGTM

**Pull request merged**